.env

```diff
@@ -9,39 +9,49 @@ INDEXER_DB_NAME=query_node_indexer
 DB_NAME=query_node_processor
 DB_USER=postgres
 DB_PASS=postgres
-DB_HOST=db
+DB_HOST=localhost
```

Comment:
I think we should actually stick with `db`, because otherwise the value in `/query-node/generated/graphql-server/.env` will vary between `db` and `localhost` during query node startup process, which seems quite fragile.

Reply:
**EDIT:** Actually nevermind this. The reason why this was occuring for me was because I:
1. Changed `DB_HOST=db` in `.env`
2. Removed `- DB_HOST=db` under `graphql-server` environment in `docker-compose.yml` as it was no longer necessary
3. Changed `DB_HOST` back to `localhost` in `.env` for some testing, but didn't add back the config removed in step `2.`, which resulted in the issue mentioned in the comment above.

Reply:
done in b3ca360396da7a1c76a6eeb4b39c118bf5f433ee